test/js/ez-navigation-hub.js

```diff
@@ -7,4 +12,128 @@ describe('ez-navigation-hub', function() {
             element.constructor
         );
     });
+    describe('properties', function () {
+        describe('`activeZoneClass`', function () {
+            it('should have a default value', function () {
+                assert.equal(element.activeZoneClass, element.getAttribute('active-zone-class'));
+            });
+
+            it('should set', function () {
```

Comment:
bad name, should be something like `it('should be set by the attribute', function () {`